hooks/RPM.js

```diff
@@ -30,20 +30,32 @@
 module.exports = function (input) {
   const { id, sentence, parts, tags } = input
 
-  const delta = {
+  return {
     updates: [
       {
         source: tags.source,
         timestamp: tags.timestamp,
+        meta: [
+          {
+            path: `propulsion.engine_${parts[1]}.pitch`,
+            value: {
+              description: `Propeller pitch in [%]`,
+              units: `%`
+             }
+          }
+        ],
         values: [
           {
             path: `propulsion.${(parts[0].toUpperCase() === 'S' ? 'shaft' : 'engine')}_${parts[1]}.revolutions`,
```

Comment:
There is an existing place in the spec for this, see https://github.com/SignalK/specification/blob/master/schemas/groups/propulsion.json

Reply:
I add only the information for pitch. The pitch is not defined and need the meta information to unit. In most cases we dont use the pitch. My last contact with a motorboat driver needs the pitch information. He has two bigger engines and want to control both maschines.

Reply:
Pitch is there. propulsion.*.drive.propeller.pitch

Reply:
Ok, I change to this path